gst/nnstreamer/tensor_split/gsttensorsplit.c

```diff
@@ -115,7 +115,7 @@ gst_tensor_split_class_init (GstTensorSplitClass * klass)
 
   parent_class = g_type_class_peek_parent (klass);
 
-  gobject_class->dispose = gst_tensor_split_dispose;
+  gobject_class->finalize = gst_tensor_split_finalize;
```

Comment:
Please explain why are you are changing the callback.

Reply:
The GStreamer destruction process is split in two phases: the first phase, executed in the dispose handler is supposed to release all references to other member objects. The second phase, executed by the finalize handler is supposed to complete the object's destruction process. 
Currently, the tensor_split has only dispose function and no finalize function.
So, I changed to finalize the tensor_split instance.


Reference link 1: https://developer.gnome.org/gobject/unstable/gobject-memory.html#gobject-memory-cycles
Reference link 2: https://github.com/ImageMagick/glib/blob/52f7289456c08d74f50d22586a86ea2827d9eec3/gobject/gobject.h#L268